lib/account/pages/about_page.dart

```diff
@@ -9,15 +12,101 @@ class AboutPage extends StatefulWidget {
 }
 
 class _AboutPageState extends State<AboutPage> {
+  PackageInfo? _packageInfo;
+
+  @override
+  void initState() {
+    super.initState();
+    PackageInfo.fromPlatform().then(
+      (value) => setState(
+        () => _packageInfo = value,
+      ),
+    );
+  }
+
   @override
   Widget build(BuildContext context) {
     return Scaffold(
       appBar: AppBar(
         title: Text(S.of(context).pageAboutTitle),
       ),
       body: Center(
-        child: SingleChildScrollView(
-          child: Text(S.of(context).appName),
+        child: CustomScrollView(
+          physics: const ClampingScrollPhysics(),
+          slivers: [
+            SliverFillRemaining(
+              hasScrollBody: false,
+              child: Padding(
+                padding: const EdgeInsets.all(10),
+                child: Column(children: [
+                  Expanded(child: Container()),
+                  Text(
+                    S.of(context).appName,
+                    style: Theme.of(context).primaryTextTheme.displaySmall,
+                  ),
+                  if (_packageInfo != null) ...[
+                    Text(
+                      S.of(context).pageAboutVersion(_packageInfo!.version),
+                      style: Theme.of(context)
```

Comment:
move the style into a variable? it's always the same

Reply:
Yes I did that, but I'd like to make a new color scheme entry for this anyways... There are many areas in the code where we use this greyed out onSurface with opacity